console/src/state/resources.rs

```diff
@@ -158,22 +166,53 @@ impl ResourcesState {
             };
 
             let id = resource.id?.id;
+            let parent_id = resource.parent_resource_id.map(|id| id.id);
+
+            let parent = strings.string(match parent_id {
+                Some(id) => parents
+                    .get(&id)
+                    .and_then(|r| r.upgrade())
+                    .map(|r| {
+                        let r = r.borrow();
+                        format!("{} ({}::{})", r.id(), r.target(), r.concrete_type())
+                    })
+                    .unwrap_or(format!("{}", id)),
+                None => "n/a".to_string(),
+            });
+
+            let parent_id = strings.string(match parent_id {
+                Some(id) => {
+                    format!("{}", id)
+                }
+                None => "n/a".to_string(),
```

Comment:
nit, take it or leave it: this could maybe be simplified to
```suggestion
            let parent_id = strings.string(parent_id
                .map(u64::to_string)
                .unwrap_or_else(|| "n/a".to_string());
            );
```